runbotics/runbotics-scheduler/src/websocket/gateway/bot.gateway.ts

```diff
@@ -71,15 +83,16 @@ export class BotWebSocketGateway implements OnGatewayDisconnect, OnGatewayConnec
     }
 
     @UseGuards(WsBotJwtGuard)
-    @SubscribeMessage(BotWsMessage.PROCESS_INSTANCE_EVENT)
     async listenForProcessInstanceEvent(
-        @ConnectedSocket() socket: BotAuthSocket,
-        @MessageBody() processInstanceEvent: IProcessInstanceEvent,
+        bot: IBot,
```

Comment:
Im worrying that UseGuards decorator won't work with this change, can u confirm it? Additionally, leave a info comment clarifying that its ws message handler

Reply:
im not sure if we need the useguard here, we are doing the validation in handle connection